g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/Interceptor/APIValidationInterceptor.java

```diff
@@ -55,8 +69,9 @@ public APIValidationInterceptor(Map<String, Object> allowedListMap) {
 	public boolean preHandle(HttpServletRequest request,
 			HttpServletResponse response, Object handler) throws Exception {
 		LOGGER.debug(request.getSession().getId());
-		String logOfUrl = "The request url is: " + request.getRequestURL();
-		String logOfQueryStr = "The request query string is: " + request.getQueryString();
+		String SingletonRequestID = getRequestId(request, this.clientRequestIds);
```

Comment:
If config.client.requestIds is empty, then don't need to call this method in order to improve performance.

Reply:
Yes, you are right. I have done the business in start of the getRequetId method